wheel/chia_rs.pyi

```diff
@@ -20,9 +20,9 @@ class Spend:
     def __repr__(self) -> str: ...
     def __richcmp__(self) -> Any: ...
     @staticmethod
-    def from_bytes(self, bytes) -> Spend: ...
+    def from_bytes(bytes) -> Spend: ...
     @staticmethod
-    def parse_rust(self, bytes) -> Tuple[Spend, int]: ...
+    def parse_rust(bytes) -> Tuple[Spend, int]: ...
```

Comment:
You might consider making this a "classmethod" rather than static methods since they generally use the class declaration.

```
def from_bytes(cls, bytes) -> Spend: ...
```